.changeset/plenty-snails-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/selectable-search-input': minor
+---
+
+When the input of the selectable search input changes, it should show the currently updated value on the input field.
```

Comment:
Perhaps we can provide just a little context:

```suggestion
We're introducing a **temporary** property to handle an internal case.
Please do not use it as it will be removed in the near future.
```

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2776/commits/1e62e9989cd8dd9e9f24cfa0f87955cf7e681ad3
Thanks